tests/check_framework/urls/path_compatibility/matched_angle_brackets.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import path
```

Comment:
Rename file to `matched_angle_brackets.py`.

Reply:
It's already is